lib/takeaway.rb

```diff
@@ -0,0 +1,18 @@
+class Takeaway
+
+  attr_reader :current_order
+
+  def initialize(menu)
+    @dishes = menu.dishes
+    @current_order = {}
+  end
+
+  def show_menu
+    @dishes
+  end
+
+  def order(dish, quantity = 1)
+    raise 'Dish not available: not part of the menu' unless @dishes.include?(dish)
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)